discovery-kubernetes-api/src/main/scala/akka/discovery/kubernetes/PodList.scala

```diff
@@ -19,7 +19,7 @@ import akka.annotation.InternalApi
 
   final case class PodSpec(containers: immutable.Seq[Container])
 
-  final case class PodStatus(podIP: Option[String])
+  final case class PodStatus(podIP: Option[String], phase: Option[String])
```

Comment:
Why an `Option`? Is it not guaranteed to be there?

(I guess the same could be said about the `podIP`, though)

Reply:
I wondered the same and preferred to be on the safe side